src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -592,7 +592,10 @@ Empty-room MEG recordings capture the environmental and recording system's
 noise.
 In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
 each experimental session.
-It is RECOMMENDED to store the empty-room recording inside a subject directory
+
+If empty-room recordings are not done prior to each individual experimental session, but
```

Comment:
"If empty-room recordings are not collected for each individual experimental session, one empty-room recording may be used with multiple subjects and/or sessions. For example, they may be collected once a day."

Reply:
thanks, I used your text.